src/applications/mhv/secure-messaging/components/ComposeForm/CategoryInput.jsx

```diff
@@ -41,6 +41,7 @@ const CategoryInput = props => {
             <VaRadioOption
               data-dd-privacy="mask"
               data-testid="compose-category-radio-button"
+              data-dd-action-name="Compose-Category-Radio-Button"
```

Comment:
Is DD picky about the naming convention here? I am wondering if possible if we are able to use non-hyphen-separated values, like a normal text instead? (that would be relevant to all instances below if that is the case) So instead of `Compose-Category-Radio-Button` have `Compose category radio button`

 And in this case in particular, would we need unique values for each category?